crypto/crypto/src/x25519.rs

```diff
@@ -256,13 +256,15 @@ impl traits::ValidCryptoMaterial for PublicKey {
 
 impl std::fmt::Display for PublicKey {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "[X25519 public key: {}]", hex::encode(self.0))
+        f.write_str(&hex::encode(&self.0))
     }
 }
 
 impl std::fmt::Debug for PublicKey {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "[X25519 public key: {}]", hex::encode(self.0))
+        f.debug_tuple("x25519::PublicKey")
+            .field(&hex::encode(&self.0))
+            .finish()
     }
```

Comment:
not clear the value add of these... but doesn't it make it asymmetric from the other crypto keys?

Reply:
actually the ed25519 looks like this

```
Ed25519PublicKey(..)
```

so this makes it consistent